rtuclient.go

```diff
@@ -276,19 +285,29 @@ func (mb *rtuSerialTransporter) Send(aduRequest []byte) (aduResponse []byte, err
 	return
```

Comment:
```suggestion
	time.Sleep(mb.frameDelay())
	return
```
is missing here, as the parser returns just after the _calculated_ last character is received instead of waiting for >=T3.5 gap in datastream.
Ref https://www.modbus.org/docs/PI_MBUS_300.pdf Page 9

**This** finally fixes the read errors with multiple devices on the bus without fixing the parser itself.

Reply:
Waiting for the gap after receive happens before the next write. See above.